src/mapgen.cpp

```diff
@@ -1499,6 +1501,11 @@ class jmapgen_computer : public jmapgen_piece
             if( target && miss ) {
                 cpu->mission_id = miss->get_id();
             }
+
+            // The default access denied message is defined in computer's constructor
+            if( !access_denied.empty() ) {
+                cpu->set_access_denied_msg( access_denied );
```

Comment:
```suggestion
                cpu->set_access_denied_msg( access_denied.translated() );
```

Reply:
I've made the change, but I just realized that it's likely better to call `translated` when printing the string instead of setting it here. Otherwise the localized string get saved along with the rest of the computer, and if you change the language of the game I think it will stay in the previous language.

Reply:
Done.